### PR TITLE
feat: Connections List Item Network Badges

### PR DESCRIPTION
**Pull request by vthomas13**
## **Description**
This PR adds Network badges to each item in the connection list. 
In future PRs, we'll make the onClick send to Nidhi's connect/disconnect flow
TODO: fix storybook

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/23397?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1978

## **Manual testing steps**

1. Open Extension
2. Make sure that Request Queueing Setting is turned on in Experimental tab
3. Connect to some Dapps
4. Go to All Permissions page and verify that the network for each connection is accurate.

### **After**

image

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

**Review by NidhiKJha — DISMISSED** [review]

**Review by darkwing — DISMISSED** [review]
This is awesome!

File: ui/selectors/selectors.js [hunk 1]
```diff
@@ -1249,7 +1252,22 @@ export const getConnectedSitesList = createDeepEqualSelector(
         }
       });
     });
+    return sitesList;
+  },
+);
 
+export const getConnectedSitesListWithNetworkInfo = createDeepEqualSelector(
```
Comment (NidhiKJha) [inline ui/selectors/selectors.js:1259]: Can we add a test for this selector?

**Review by NidhiKJha — CHANGES_REQUESTED** [review]
Looks amazing. But can we add a test for this selector?

**Review by NidhiKJha — APPROVED** [review]

**Review by darkwing — APPROVED** [review]
Well done!

**Pull request merged**